src/main/webapp/SA.html

```diff
@@ -70,6 +70,171 @@
+                                                    <ul>
+                                                        <li>A high degree of multi-tasking and time management
+                                                            capability
+                                                        </li>
+                                                        <li>Good communication skills in English and Sinhala or Tamil
+                                                        </li>
+                                                        <li>Exemplary leadership qualities
+                                                        </li>
+                                                    </ul>
+                                                </div>
+                                            </div>
+                                        </div>
+                                        <div class="py-3 card-toggle">
+                                            <div class="" id="headingFive">
+                                                <h5 class="mb-0">
+                                                    <button
+                                                            class="btn btn-link w-100 text-dark text-left collapsed panel text-capitalize"
+                                                            type="button"
+                                                            data-toggle="collapse"
+                                                            data-target="#collapseFive"
+                                                            aria-expanded="false"
+                                                            aria-controls="collapseFive"
+                                                    >
+                                                        When can I apply?
+                                                        <div class="text-primary"><i
+                                                                class="plusMinusSign fa fa-plus float-right"></i></div>
+                                                    </button>
+                                                </h5>
+                                            </div>
+                                            <div
+                                                    id="collapseFive"
+                                                    class="collapse"
+                                                    aria-labelledby="headingFive"
+                                                    data-parent="#accordionExample"
+                                            >
+                                                <div class="card-body opacity-8 text-left">
+                                                    We recruit people throughout the entire year.
+                                                    Submit your application using the Apply Now button below.
```

Comment:
This line should be changed as "Apply Now button above".
It was my mistake. I did not notice this when I created the issue.